### PR TITLE
Watch assets in separate container

### PR DESCRIPTION
**Pull request by efischer19**

### CODE REVIEW

**Comment by efischer19** [comment]
This kind of works, but tests are failing due to some sort of "no space on device" issue. I'll keep poking next week, but feel free to take a look at what's already here.

**Review by macdiesel — APPROVED** [review]
You can ignore the disk space error and merge this if the tests pass locally.

File: docker-compose.yml [hunk 1]
```diff
@@ -149,6 +149,19 @@ services:
     volumes:
       - edxapp_lms_assets:/edx/var/edxapp/staticfiles/
 
+  lms_watcher:
+    command: bash -c 'cd /edx/app/edxapp/edx-platform && source ../edxapp_env && while true; do paver watch_assets; sleep 2; done'
+    container_name: edx.devstack.lms_watcher
+    depends_on:
+      - lms
+    stdin_open: true
```
Comment (gsong) [inline docker-compose.yml]: Is there a reason for needing `stdin` and `tty`?

File: docker-compose.yml [hunk 2]
```diff
@@ -149,6 +149,19 @@ services:
     volumes:
       - edxapp_lms_assets:/edx/var/edxapp/staticfiles/
 
+  lms_watcher:
+    command: bash -c 'cd /edx/app/edxapp/edx-platform && source ../edxapp_env && while true; do paver watch_assets; sleep 2; done'
+    container_name: edx.devstack.lms_watcher
+    depends_on:
+      - lms
```
Comment (gsong) [inline docker-compose.yml]: Does this service really depend on `lms` service starting?

Comment (efischer19) [reply docker-compose.yml]: Yeah you're right - without the lms process we can still watch assets, it's just kind of pointless b/c nothing is consuming them. No reason to force that dependence.

File: docker-compose.yml (hunk 1 above)
Comment (efischer19) [reply docker-compose.yml]: Nope, I just copy-pasted :)

I've tested and things look fine with this removed, I'll update.

**Comment by efischer19** [comment]
Failure is the disk space issue again, merging.

**Pull request merged**